ldms/src/test/Makefile.am

```diff
@@ -39,4 +39,17 @@ test_metric_SOURCES = test_metric.c
 test_metric_LDADD = -lldms
 test_metric_LDFLAGS = $(AM_LDFLAGS) -pthread -lm
 
+install-data-hook: ldms-static-test-list.sh ldms-run-static-tests.sh ldms-static-test-bypass
+	$(MKDIR_P) $(DESTDIR)$(pkglibdir)
+	$(INSTALL) -m 755 ldms-static-test-list.sh $(DESTDIR)$(pkglibdir)
+	$(INSTALL) -m 755 ldms-run-static-tests.sh $(DESTDIR)$(pkglibdir)
+	$(INSTALL) -m 644 ldms-static-test-bypass $(DESTDIR)$(pkglibdir)
+
+uninstall-hook:
+	rm $(DESTDIR)$(pkglibdir)/ldms-static-test-list.sh
+	rm $(DESTDIR)$(pkglibdir)/ldms-run-static-tests.sh
+	rm $(DESTDIR)$(pkglibdir)/ldms-static-test-bypass
+
+installcheck-local: ldms-run-static-tests.sh
```

Comment:
ldms-run-static-tests.sh should be in the body rather than a prerequisite target.